packages/flutter_tools/lib/src/fuchsia/fuchsia_build.dart

```diff
@@ -98,7 +98,8 @@ Future<void> _genSnapshot(
 
   final List<String> command = <String>[
     genSnapshot,
-    '--no_causal_async_stacks',
```

Comment:
I'm not sure if this matters...

Reply:
If dashes work for other flags, they should also work for these flags.